registry/extension/oras/artifactmanifesthandler.go

```diff
@@ -78,6 +79,17 @@ func (amh *artifactManifestHandler) verifyManifest(ctx context.Context, dm Deser
 		errs = append(errs, distribution.ErrManifestVerification{errors.New("artifactType invalid")})
 	}
 
+	if dm.MediaType() == "" {
+		errs = append(errs, distribution.ErrManifestVerification{errors.New("mediaType invalid")})
+	}
+
+	if createdAt, ok := dm.Annotations()[createAnnotationName]; ok {
+		_, err := time.Parse(time.RFC3339, createdAt)
+		if err != nil {
+			errs = append(errs, distribution.ErrManifestVerification{errors.New("failed to parse created time: " + err.Error())})
+		}
+	}
+	// TODO: check that created annotation has correct format
```

Comment:
What check do you plan to do in addition to `time.Parse`?

Reply:
the TODO was actually referring to the extra time.parse check I added in the last commit. I need to go back and remove the comment